src/applications/disability-benefits/686c-674/tests/e2e/00-child-marriage-report-death-stepchild-left-child-stopped-school.e2e.spec.js

```diff
@@ -36,19 +36,19 @@ const runTest = E2eHelpers.createE2eTest(client => {
     ],
     testData.data,
   );
-  client.click('button[id="2-continueButton"]');
+  client.click('button[id="4-continueButton"]');
```

Comment:
To make this more robust, you could use this selector: `'[button[id$="-continueButton"]'` (unless you can't)

See https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Attribute_selectors#Substring_matching_selectors for more

Reply:
Tried this and unfortunately it doesn't seem to work :(

Reply:
<img title=":_ onion_head_cry :" src="https://cloud.githubusercontent.com/assets/136959/13371449/f9137e10-dceb-11e5-8c0b-24e9eb565187.gif">